src/lib.rs

```diff
@@ -108,9 +115,13 @@ mod backtrace;
 pub use symbolize::{resolve, Symbol, SymbolName};
 mod symbolize;
 
+#[cfg(feature = "std")]
 pub use capture::{Backtrace, BacktraceFrame, BacktraceSymbol};
+#[cfg(feature = "std")]
 mod capture;
 
+mod types;
```

Comment:
I thin kwe'll want to be sure to reexport the enum in this modules at the top-level as well